MapboxNavigation/FeedbackItem.swift

```diff
@@ -81,6 +81,10 @@ public extension FeedbackType {
             return NSLocalizedString("ROUTE_QUALITY_ROAD_MISSING_FROM_MAP_FEEDBACK", bundle: .mapboxNavigation, value: "Road missing from map", comment: "Specifc route feedback that route contained road not shown on map.")
         case .roadClosure(.other):
             return NSLocalizedString("ROAD_CLOSURE_OTHER_FEEDBACK", bundle: .mapboxNavigation, value: "Other", comment: "Specific route feedback that a road closure type was encountered but not listed as a choice.")
+        case .positioning(.none):
+            return NSLocalizedString("POSITIONING", bundle: .mapboxNavigation, value: "Positioning", comment: "General category of route feedback where user position is incorrect.")
+        case .positioning(.userPosition):
+            return NSLocalizedString("POSITIONING", bundle: .mapboxNavigation, value: "User Position", comment: "Specific route feedback that user positioning is incorrect.")
```

Comment:
These two strings should have different keys.

Remember to run scripts/extract_localizable.sh to update the Localizable.strings file. Unfortunately, there won’t be much time to get translations of this string, but we’ll pull in any missing translations in v1.1.